ush/config_defaults.yaml

```diff
@@ -2046,268 +2078,290 @@ task_get_obs_ndas:
   MAXTRIES_GET_OBS_NDAS: 1
 
 #----------------------------
-# VX_GRIDSTAT config parameters
+# run_MET_GridStat_vx_APCP01h config parameters
 #-----------------------------
-task_run_vx_gridstat:
-  TN_VX_GRIDSTAT: "run_gridstatvx"
-  NNODES_VX_GRIDSTAT: 1
-  PPN_VX_GRIDSTAT: 1
-  MEM_VX_GRIDSTAT: 2G
-  WTIME_VX_GRIDSTAT: 02:00:00
-  MAXTRIES_VX_GRIDSTAT: 2
+task_run_MET_GridStat_vx_APCP01h:
+  TN_RUN_MET_GRIDSTAT_VX_APCP01h: "run_MET_GridStat_vx_APCP01h"
+  NNODES_RUN_MET_GRIDSTAT_VX_APCP01H: 1
+  PPN_RUN_MET_GRIDSTAT_VX_APCP01H: 1
+  MEM_RUN_MET_GRIDSTAT_VX_APCP01H: 2G
+  WTIME_RUN_MET_GRIDSTAT_VX_APCP01H: 02:00:00
+  MAXTRIES_RUN_MET_GRIDSTAT_VX_APCP01H: 2
 
 #----------------------------
-# VX_GRIDSTAT_REFC config parameters
+# run_MET_GridStat_vx_APCP03h config parameters
 #-----------------------------
-task_run_vx_gridstat_refc:
-  TN_VX_GRIDSTAT_REFC: "run_gridstatvx_refc"
-  NNODES_VX_GRIDSTAT: 1
-  PPN_VX_GRIDSTAT: 1
-  MEM_VX_GRIDSTAT: 2G
-  WTIME_VX_GRIDSTAT: 02:00:00
-  MAXTRIES_VX_GRIDSTAT_REFC: 2
+task_run_MET_GridStat_vx_APCP03h:
```

Comment:
To be consistent with the modified all lowercase ex-script name of the task, this also needs to be all in lowercase.

Reply:
@danielabdi-noaa Changed these as well.